natlas-server/migrations/versions/3fb710fe0fe1_configitem_table.py

```diff
@@ -16,9 +16,18 @@
 depends_on = None
 
 
+def get_default_configs():
+    return [
+        {"id": 1, "name": "LOGIN_REQUIRED", "type": "bool", "value": "True"},
+        {"id": 2, "name": "REGISTER_ALLOWED", "type": "bool", "value": "False"},
+        {"id": 3, "name": "AGENT_AUTHENTICATION", "type": "bool", "value": "True"},
+        {"id": 4, "name": "CUSTOM_BRAND", "type": "string", "value": ""},
+    ]
+
+
 def upgrade():
```

Comment:
Why are we adding these changes to an existing migration? Aren't migrations supposed to be append-only and we should create a new migration? If my DB is already past 3fb710fe0fe1 how does Flask know to run this?

Reply:
While I would normally agree to this, they should also be atomic in nature and having a database table created 18 migrations ago but having the values not populated until just now creates a whole different set of problems now that the population code has been removed from app init.

Reply:
If your database is already past `3fb710fe0fe1` then you really wouldn't want me to push a change that resets the values in your database to the defaults anyways. I think they belong with the migrations that create the corresponding tables, however we can also make a new migration that checks the database for any existing values and if they already exist, don't modify their value.

Reply:
> they should also be atomic in nature and having a database table created 18 migrations ago but having the values not populated until just now creates a whole different set of problems now that the population code has been removed from app init.

Migrations have to represent the path that a DB went through. Migrations should be run from `$CURRENT_STATE` to latest, thus I don't see how it's a concern if the table is created separately from being filled. This is fundamentally the path that the DB took while developing.

While this happens to work in this case, I would strongly urge us to avoid these kinds of changes in the future.

Reply:
I agree, I think all future additions must be additive. I felt dirty retroactively modifying them. Let #401 mark future changes to existing migrations as "Unauthorized".